components/Ripple/src/MDCStatefulRippleView.m

```diff
@@ -75,11 +75,6 @@ - (void)commonMDCStatefulRippleViewInit {
 
 - (UIColor *)rippleColorForState:(MDCRippleState)state {
   UIColor *rippleColor = _rippleColors[@(state)];
-  if (rippleColor == nil && (state & MDCRippleStateDragged) != 0) {
```

Comment:
do you prefer the fallback to selected+X and dragged+X to fallback to normal as opposed to selected and dragged? Did you make sure this didn't break visually Cards and other components?

Reply:
I changed my mind about this issue once I started working with state combinations in Themers.  You would expect that color defaults for ripple will behave the same as for background and title. 

I ran a global presubmit on this and all other changes in these PRs. Does that verify it enough?

Reply:
its less the global, but more verifying that the Cards with Ripple examples have the correct colors and that they don't rely on this logic. A case I could see happening is having Selected+Dragged fallback to normal where it makes more sense to it to fallback to dragged or selected. Also, bear in mind that Ripple doesn't really have a notion of colors for Normal as Normal basically means there is no ripple. So I think it is somewhat different from background and title

Reply:
I'm not sure what else needs to be checked. The catalog looks good. All tests pass. Anyway - I removed it now so we can land the PRs.